app/helpers/application_helper.rb

```diff
@@ -589,8 +589,9 @@ def creation_id_for_css_classes(creation)
   # External works are not created by users, so we can skip this.
   # TODO: AO3-6132 to add creator ids to series blurbs.
   def creator_ids_for_css_classes(creation)
-    return [] unless creation.is_a?(Work)
-    return [] if creation.anonymous? || creation.unrevealed?
+    return [] unless %w[Series Work].include?(creation.class.name)
+    return [] if creation.anonymous?
+    return [] if creation.is_a?(Work) && creation.unrevealed?
```

Comment:
Hmm, maybe it isn't. I'll have to see, since I can't remember.

Reply:
> Because you can see the creator names in the blurb, we should add classes for them as well.

Is this a TODO for when we eventually need to check for unrevealed series, or is it a description of what the method is doing now?

Reply:
Both, really. It currently adds classes to series blurbs (but they're not visible anywhere) and it's a reminder to make sure we add them to the blurbs of bookmarked series as well.